src/NServiceBus.Metrics.ServiceControl/ServiceControlReporting/EndpointMetadata.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace NServiceBus.Metrics.ServiceControl.ServiceControlReporting
+{
+    class EndpointMetadata
+    {
+        string localAddress;
+
+        public EndpointMetadata(string localAddress)
+        {
+            this.localAddress = localAddress;
+        }
+
+        public string ToJson()
+        {
+            return SimpleJson.SerializeObject(new
+            {
+                Version = 1,
```

Comment:
Is this a version for the protocol versioning? If yes, I'd add a comment in here.

Reply:
This is not the protocol version but version of the plugin we are running in. Comment added.